updates/Evan Limanto.md

```diff
@@ -0,0 +1,8 @@
+# Evan Limanto - weekly updates
+
+## 02-24-2017
+
+- **Past week:** Worked on 3 bug fix PRs, all tests passing and awaiting review
```

Comment:
Please add the URLs here.  This document is really meant as a  kind of log book that we can refer back to at the end of the program.